src/index.ts

```diff
@@ -148,15 +154,13 @@ export default async function createCheck({
     },
   };
 
-  const run = await octokit.checks
-    .create(check)
-    .catch(async (error) => {
-      // eslint-disable-next-line no-console
-      console.log('Failed to create check with error:', error);
-      const PRE_HEAD = (await execa('git', ['rev-parse', 'HEAD^1'])).stdout;
-      // Retrying against parent commit
-      return octokit.checks.create({ ...check, head_sha: PRE_HEAD })
-    });
+  const run = await octokit.checks.create(check).catch(async (error) => {
+    // eslint-disable-next-line no-console
+    console.log('Failed to create check with error:', error);
+    const PRE_HEAD = (await execa('git', ['rev-parse', 'HEAD^1'])).stdout;
+    // Retrying against parent commit
+    return octokit.checks.create({ ...check, head_sha: PRE_HEAD });
+  });
```

Comment:
Just reformatted this block or what? Could revert for a cleaner diff.

Reply:
It was autoformatted, I assumed default prettier took over. I could revert this.